src/components/Todo.js

```diff
@@ -0,0 +1,40 @@
+function Todo(vname) {
+  let name = vname;
+  let description = null;
+  let date = null;
+  let priority = 'default';
+  let checked = false;
+  // const timeStamp = ;
```

Comment:
- [ ] Please remove this comments. There shouldn't be comments in your code. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))